include/SampleBuffer.h

```diff
@@ -37,6 +37,7 @@
 #include "lmms_math.h"
 #include "shared_object.h"
 #include "MemoryManager.h"
+#include "PathUtil.h"
```

Comment:
I think you may remove the `#include` statement and add it in source files where needed.

Reply:
As a local include inside the functions that use it? Or how do you mean?

Reply:
At the beginning of a source code that requires `PathUtil.h`.

Reply:
Oh, wait. I should include it in SampleBuffer.cpp instead of SampleBuffer.h right?

Reply:
`audio_file_processor.cpp` needs it as well(previously satisfied via `SampleBuffer.h`).

Reply:
Fixed, finally.